initramfs/src/image.rs

```diff
@@ -126,11 +152,11 @@ impl Image {
             }
         }
 
-        let mut init_file =
-            File::open(init_path).map_err(|e| anyhow!(e).context("Failed to open init file"))?;
-        let mut agent_file =
-            File::open(agent_path).map_err(|e| anyhow!(e).context("Failed to open init file"))?;
-        let mut agent_config_file = File::open(agent_config_path)
+        let mut init_file = <Handler>::open(init_path)
```

Comment:
```suggestion
        let mut init_file = Handler::open(init_path)
```

Probably need to run `rustfmt` after that, it's probably not going to format it the same way.